README.md

```diff
@@ -59,21 +59,28 @@ The following table lists all work streams and links to their home directory and
 - Clone this repository
 - Clone this repository's submodules:
 
-```bash
-git submodule update --init --recursive
-```
+    ```bash
+    git submodule update --init --recursive
+    ```
 
 ### Install Dependencies
 
 - Install [Go](https://golang.org/doc/install) (Flow supports Go 1.13 and later)
 - Install [CMake](https://cmake.org/install/), which is used for building the crypto library
 - Install [Docker](https://docs.docker.com/get-docker/), which is used for running
   a local network and integration tests
-- All remaining dependencies can be installed automatically:
-
-```bash
-make install-tools
-```
+- Make sure the `GOPATH` and `GOBIN` environment variables are set, and `GOBIN` is added to your path:
+
+    ```bash
+    export GOPATH=$(go env GOPATH)
```

Comment:
This will work the first run, but in general I think they will need to add this to their shell profile so it is persisted. Maybe we can add a note to do that as well and link to [the docs](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable)